src/Symfony/Component/Uid/Factory/TimeBasedUuidFactory.php

```diff
@@ -27,6 +30,7 @@ public function __construct(string $class, Uuid $node = null)
 
     public function create(\DateTimeInterface $time = null): Uuid&TimeBasedUidInterface
     {
+        /** @var class-string<UuidV1|UuidV6|UuidV7> $class */
```

Comment:
Shouldn't we annotate this type to the `$class` property instead?

Reply:
Done